server/interview/views.py

```diff
@@ -63,6 +63,35 @@ def is_valid_availability(availability):
     )
 
 
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAdminUser
+from django.utils import timezone
+from datetime import timedelta
+
+@api_view(['GET'])
+@permission_classes([])
+def get_signup_data(request):
```

Comment:
Maybe change to class-based view for consistency throughout the codebase?

Reply:
changed, however I don't mind using a mix of class and function based views